src/liquid/auth/authentication_form.html

```diff
@@ -1,39 +1,32 @@
-<form id="form_authenticate" hx-post="/authenticate" hx-target="this" hx-swap="outerHTML">
+<form class="form" id="form_authenticate" hx-post="/authenticate" hx-target="this" hx-swap="outerHTML">
   <!-- want to use tiles, they are super weird with forms in them -->
-  <div class="box" style="margin: 20px; padding: 50px;">
-    {% if errors %}
-      <span class='tag is-warning is-overlay'>Login Failed. Please Try Again</span>
-    {% endif %}
-
-    <div class="field">
-      <label class="label">Instance</label>
-      <div class="control">
-        <div class="select is-fullwidth">
-          <select name="domain">
-              {% for domain in domains %}
-                <option value="{{ domain.domain }}">{{ domain.friendly }}</option>
-              {% endfor %}
-          </select>
-        </div>
+  <div class="">
+    <div class="">
+      <label>Instance</label>
+      <div>
+        <select class="custom-select" name="domain">
+          {% for domain in domains %}
+          <option value="{{ domain.domain }}">{{ domain.friendly }}</option>
+          {% endfor %}
+        </select>
       </div>
     </div>
-    <div class="field">
-      <label class="label">Credentials</label>
-      <div class="control">
-          <input name="username" class="input" type="text" placeholder="username">
+    <div>
+      <label>Credentials</label>
+      <div class="input-fields">
+        <div><input name="username" type="text" placeholder="Username"></div>
+        <div><input name="password" type="password" placeholder="Password"></div>
       </div>
     </div>
-    <div class="field">
-        <div class="control">
-            <input name="password" class="input" type="password" placeholder="password">
-        </div>
+    {% if errors %}
+    <div class="error center">
+      <span>Login Failed. Please Try Again</span>
     </div>
-    <div class="field is-grouped is-grouped-right">
-      <p class="control">
-        <button class="button is-primary">
-          Sign In
-        </button>
-      </p>
+    {% endif %}
+    <div class = "">
+      <button class=" login-btn">
```

Comment:
When I click `login`, it sometimes takes like 4-5 seconds before I see an error.
When I retry the login, the error remains and there is no indication that anything is happen. I find this state a bit difficult - waiting, with an error visible, with no indication anything is happening.
Can we do something?  Maybe hx-disable is enough (feedback above). Maybe we could do something with [hx-indicator](https://htmx.org/attributes/hx-indicator/) like in https://github.com/medic/cht-user-management/pull/142?